### ISSUE TITLE
Create spaces access key via provider

### ISSUE THREAD

**Issue by unredundant** [digitalocean/terraform-provider-digitalocean#880]
Essentially reopening #559

In my personal case, I would like to create a bucket that is used purely for Thanos storage, provision thanos itself via helm, and have access keys for the bucket programmatically provisioned and added to the helm yaml values all inside terraform.

**Comment by pzduniak** [comment]
I second this... looks like it's not supported in the API?

https://github.com/digitalocean/doctl/issues/936

**Comment by andrewsomething** [comment]
That's correct. This can not currently be done in Terraform as the API does not support it. We've passed the feature request on to the Spaces team here internally.

**Comment by joerx** [comment]
Spaces API doesn't seem to be part of the regular API, instead you're supposed to use the SDKs. But that doesn't include any way to create credentials (since in AWS that would be part of IAM). For a "developer friendly" service this is a pretty big gap.

**Comment by SayHeyD** [comment]
+1 would be great to be able to do this

**Comment by ben-pr-p** [comment]
This is also a big whole in my infrastructure automation - would love a solution here!

**Comment by yurymuski** [comment]
any upd?

**Comment by Am6puk** [comment]
any updates?

**Comment by dart-ops** [comment]
any updates?
